windows/ReactNativePSPDFKit/ReactNativePSPDFKit/PDFViewPage.xaml.cs

```diff
@@ -0,0 +1,32 @@
+﻿using PSPDFKit;
+using System;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace ReactNativePSPDFKit
+{
+    /// <summary>
+    /// A webview which will hold the pdf render
+    /// </summary>
+    public sealed partial class PDFViewPage : Page
+    {
+
+        API _API;
+        string _license;
+
+        public PDFViewPage(API api, string license)
+        {
+            _API = api;
+            _license = license;
+            InitializeComponent();
+        }
+
+        private void PDFView_NavigationStarting(WebView sender, WebViewNavigationStartingEventArgs args)
+        {
+            _API.InitializeWithWebView(_license, sender);
```

Comment:
This now requires `css` argument. But I guess you want to stay up to date with the latest published beta which doesn't?

Reply:
Yeah. I'm going to have to leave this for now as the customer will have the beta release. This could get a little tricky. Maybe have to schedule an update period.